core/templates/dev/head/pages/topics_and_skills_dashboard/topics_list_directive.html

```diff
@@ -1,97 +1,94 @@
-          <p ng-if="key === 'subtopic_count'" translate="I18N_DASHBOARD_TABLE_HEADING_SUBTOPICS"></p>
-          <p ng-if="key === 'skill_count'" translate="I18N_DASHBOARD_TABLE_HEADING_SKILLS"></p>
-          <p ng-if="key === 'canonical_story_count'" translate="I18N_DASHBOARD_TABLE_HEADING_CANONICAL_STORIES"></p>
-        </th>
-      </tr>
-      <tr ng-repeat="topic in topicSummaries()"
-          class="list-item">
-        <td>
-          <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
-            <span> <[topic.name]> </span>
-          </a>
-        </td>
-        <td>
-          <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
-            <span> <[topic.subtopic_count]> </span>
-          </a>
-        </td>
-        <td>
-          <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
-            <span> <[topic.total_skill_count]> </span>
-            <span class="additional-text" ng-if="topic.uncategorized_skill_count > 0"> ( <[topic.uncategorized_skill_count]> uncategorized ) </span>
-          </a>
-        </td>
-        <td>
-          <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
-            <span> <[topic.canonical_story_count]> </span>
-          </a>
-        </td>
-      </tr>
-    </table>
-  </md-card>
-</div>
+<md-card layout="row" class="list-view-item">
+  <table class="dashboard-table">
+    <colgroup>
+      <col style="width: 28%;">
+      <col style="width: 24%;">
+      <col style="width: 24%;">
+      <col style="width: 24%;">
+    </colgroup>
+    <tr>
+      <th ng-repeat="key in TOPIC_HEADINGS"
+          class="dashboard-table-headings">
+        <p ng-if="key === 'name'"> Topic Name </p>
+        <p ng-if="key === 'subtopic_count'"> Subtopics </p>
+        <p ng-if="key === 'skill_count'"> Skills </p>
+        <p ng-if="key === 'canonical_story_count'"> Canonical Stories </p>
+      </th>
+    </tr>
+    <tr ng-repeat="topic in getTopicSummaries()"
+        class="list-item">
+      <td>
+        <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
+          <span> <[topic.name]> </span>
+        </a>
+      </td>
+      <td>
+        <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
+          <span> <[topic.subtopic_count]> </span>
+        </a>
+      </td>
+      <td>
+        <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
+          <span> <[topic.total_skill_count]> </span>
+          <span class="additional-text" ng-if="topic.uncategorized_skill_count > 0"> ( <[topic.uncategorized_skill_count]> uncategorized ) </span>
+        </a>
+      </td>
+      <td>
+        <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="list-summary">
+          <span> <[topic.canonical_story_count]> </span>
+        </a>
+      </td>
+    </tr>
+  </table>
+</md-card>
 
 <style>
-  .topics-list .list-view-item {
+  topics-list .list-view-item {
```

Comment:
Again do you need list-view-item?

Reply:
Same here, when I removed .list-view-item, the styling was removed.